src/Microsoft.DotNet.Cli.Utils/DotnetFiles.cs

```diff
@@ -21,7 +21,7 @@ internal static class DotnetFiles
         /// </summary>
         public static string VersionFile => Path.GetFullPath(Path.Combine(SdkRootFolder, ".version"));
 
-        internal static DotnetVersionFile VersionFileObject
+        public static DotnetVersionFile VersionFileObject
```

Comment:
@livarcocc I believe this is useful outside of this dll too - why InternalsVisibleTo?